pkg/log/log.go

```diff
@@ -0,0 +1,40 @@
+// Package log provides a global interface to logging functionality
+package log
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+)
+
+type id struct{}
```

Comment:
I think it would be more obvious to use strings for context value keys, e.g "crio/id". Unless it's for performance reasons, in which case this probably needs a comment.

Reply:
I think that using strings would let us attach a container ID as well?

Reply:
Golint complains when I use a string there:
```
should not use basic type string as key in context.WithValue
```

I think its about possible collisions we have to avoid. :)

Reply:
On interesting. Well that won't preclude us adding a container ID in the future so 👍